changelog.d/pa-1006.added

```diff
@@ -0,0 +1,8 @@
+Matching: Qualified names written as patterns can now match valid instances of
+identifiers which lie underneath a wildcard import. For instance, in Python,
+we could write the pattern `A.B.C.x`, and match the usage in the program
+```
+from A.B import *
+
+foo(C.x)
```

Comment:
We need a ticket in https://linear.app/semgrep/project/documentation-93e79895033e to remember to document this feature.

Reply:
https://linear.app/semgrep/project/documentation-93e79895033e